packages/core/src/tracing/instrumentation/database/couchbase.js

```diff
@@ -493,3 +546,8 @@ function getBucketType(c, n) {
     return bucketType;
   };
 }
+
+// converts the operation into query format in uppercase
+function camelCaseToUpperWords(op) {
```

Comment:
suggestion: Use this helper function for other use cases as well. There is a couple of `toUpperCase` in the instrumentation.

Reply:
updated!